src/main/java/frc/robot/Robot.java

```diff
@@ -44,15 +52,28 @@ public void robotPeriodic() {
     // and running subsystem periodic() methods.  This must be called from the robot's periodic
     // block in order for anything in the Command-based framework to work.
 
+    Threads.setCurrentThreadPriority(true, 99);
```

Comment:
Adjusting thread priority is not a change to make without really knowing more about the impact.